dask/compatibility.py

```diff
@@ -151,6 +154,10 @@ def _make_reraise():
     reraise = _make_reraise()
     del _make_reraise
 
+    eval(compile(("def exec_(codestr, glbls):\n"
+                  "    exec codestr in glbls\n"),
+                  "<_exec>", "exec"))
```

Comment:
This was removed in #3979 before merging, you may want to rebase(?) with master to squash out the changes since then.

Reply:
Removed.  I also tried rebasing but decided against it.  Too many commits.  I'll probably merge things in and then verify that nothing odd is in the diff

Reply:
I mean merge master into this branch